tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+import dataset
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.orm import sessionmaker
+import processors.base.config as config
+
+# Make fixtures available to all tests
+
+from tests.fixtures.files import file_fixture
+from tests.fixtures.trials import trial
+from tests.fixtures.sources import nct_source, fda_source
+from tests.fixtures.fda_approvals import fda_approval
+from tests.fixtures.fda_applications import fda_application
+from tests.fixtures.organizations import organization
+
+
+def teardown_database(engine):
+    metadata = MetaData(bind=engine)
+    metadata.reflect()
+    metadata.drop_all()
+
+
+def create_test_database(source_database_url, test_database_url):
+    engine = create_engine(source_database_url)
+    test_engine = create_engine(test_database_url)
+    metadata = MetaData()
+    metadata.reflect(engine)
+    metadata.create_all(test_engine)
+    return test_engine
```

Comment:
I don't think this approach will work on test environments like Travis, as they don't have access to a "regular" database. In that case, we need to generate the schema dumps locally and commit them in.